.gitignore

```diff
@@ -4,9 +4,9 @@ logs/
 scripts/
 
 .idea
+.DS_Store
 
 edr.log
-integration_tests/data
```

Comment:
Not sure if we should upload all the data into the repo.
This is better because it creates clear fixtures.
The only downside is that `dbt deps` downloads the entire repo.